docusaurus.config.ts

```diff
@@ -81,6 +81,8 @@ const config: Config = {
   ],
 
   baseUrl: "/",
+  // For GitHub Pages, this value must be defined.
+  trailingSlash: false,
```

Comment:
I'm pretty sure this should be true.  It exhibits the same errors locally when set to false.

Reply:
Initially, I also thought that `true` is the correct one. However, in the development environment, the error reoccurred for `true` value. I'll create a static version with both values on my server to see how it behaves.

Reply:
It seems that disabling the `trailingSlash` results in a different type of output. I think setting it to 'true' doesn't solve the issue, but we can give it a try.

When it `false`: https://opentofu-preview-0871e318.nf.hu/

```console
total 580
drwxrwxrwx 1 ben ben  4096 Jan 19 15:56 .
drwxrwxrwx 1 ben ben  4096 Jan 19 15:55 ..
-rwxrwxrwx 1 ben ben 43759 Jan 19 15:56 404.html
-rwxrwxrwx 1 ben ben  1467 Jan 19 15:56 OpenTofu-Events-Calendar.ics
drwxrwxrwx 1 ben ben  4096 Jan 19 15:56 announcement
drwxrwxrwx 1 ben ben  4096 Jan 19 15:56 assets
drwxrwxrwx 1 ben ben  4096 Jan 19 15:56 blog
-rwxrwxrwx 1 ben ben 56612 Jan 19 15:56 blog.html
drwxrwxrwx 1 ben ben  4096 Jan 19 15:56 docs
-rwxrwxrwx 1 ben ben 45884 Jan 19 15:56 docs.html
-rwxrwxrwx 1 ben ben 60334 Jan 19 15:56 faq.html
drwxrwxrwx 1 ben ben  4096 Jan 19 15:56 favicons
drwxrwxrwx 1 ben ben  4096 Jan 19 15:56 fork
drwxrwxrwx 1 ben ben  4096 Jan 19 15:56 img
-rwxrwxrwx 1 ben ben 71466 Jan 19 15:56 index.html
-rwxrwxrwx 1 ben ben 51062 Jan 19 15:56 manifesto.html
-rwxrwxrwx 1 ben ben   593 Jan 19 15:56 opensearch.xml
-rwxrwxrwx 1 ben ben 49010 Jan 19 15:56 privacy.html
-rwxrwxrwx 1 ben ben 45064 Jan 19 15:56 registry.html
-rwxrwxrwx 1 ben ben 52020 Jan 19 15:56 search.html
-rwxrwxrwx 1 ben ben 39929 Jan 19 15:56 sitemap.xml
drwxrwxrwx 1 ben ben  4096 Jan 19 15:56 slack
-rwxrwxrwx 1 ben ben 55690 Jan 19 15:56 supporters.html
```

When it `true`: https://opentofu-preview-8a0a1266.nf.hu/ - Generates 404

```console
total 160
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 .
drwxrwxrwx 1 ben ben  4096 Jan 19 15:58 ..
-rwxrwxrwx 1 ben ben 43790 Jan 19 15:59 404.html
-rwxrwxrwx 1 ben ben  1467 Jan 19 15:59 OpenTofu-Events-Calendar.ics
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 announcement
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 assets
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 blog
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 docs
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 faq
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 favicons
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 fork
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 img
-rwxrwxrwx 1 ben ben 71499 Jan 19 15:59 index.html
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 manifesto
-rwxrwxrwx 1 ben ben   593 Jan 19 15:59 opensearch.xml
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 privacy
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 registry
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 search
-rwxrwxrwx 1 ben ben 40239 Jan 19 15:59 sitemap.xml
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 slack
drwxrwxrwx 1 ben ben  4096 Jan 19 15:59 supporters
```

Reply:
Okay... :) So the situation is that further testing is needed. The `false` version doesn't generate `index.html` in build, so internal links work, but direct hits don't. Currently the best option is `true`.

Reply:
Part of the problem is that https://opentofu.org/docs/intro/migration is redirected to https://opentofu.org/docs/intro/migration/ automatically so that is the version that users will land on.  I think we need to follow that convention.

Reply:
I'll change the value to `true`.

Reply:
I'll keep an eye out for cases where links might need to be updated and can handle that.

Reply:
It might also be worth checking how it works with absolute URL.

https://github.com/opentofu/opentofu/blob/main/website/docs/intro/migration.mdx#L40

Reply:
Looks like that's working just fine on the deployed instance